packages/nx/src/utils/plugins/plugin-capabilities.ts

```diff
@@ -99,19 +98,18 @@ export async function getPluginCapabilities(
 
 async function tryGetModule(
   packageJson: PackageJson,
-  workspaceRoot: string,
-  projects: Record<string, ProjectConfiguration>
-): Promise<RemotePlugin | null> {
+  workspaceRoot: string
+): Promise<NxPlugin | null> {
   try {
     return packageJson.generators ??
       packageJson.executors ??
       packageJson['nx-migrations'] ??
       packageJson['schematics'] ??
       packageJson['builders']
-      ? await loadRemoteNxPlugin(packageJson.name, workspaceRoot)
+      ? (await loadPlugin(packageJson.name, workspaceRoot)).plugin
```

Comment:
See earlier lengthy comment. If this is only reason to expose loading a plugin on the main thread, lets just not. Its very little benefit to doing it and introduces all of the issues associated with loading modules during this stuff. Let's see about deferring to a package.json field like everything else.

Reply:
It's not worth starting up a worker for this at the moment. At least for now, it's fine to require it just for metadata

Reply:
I hear you, I'm not necessarily suggesting we start workers up, but if not then let's just drop support for list mentioning graph stuff for the time being and we can add that support back in later via a package json Metadata field or similar.